### PR TITLE
Improve README and add okteto manifest

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/14079

Readme based on https://github.com/giantswarm/aws-operator/blob/master/README.md
Okteto setup based on https://github.com/giantswarm/aws-operator/pull/2827

### CODE REVIEW

File: helm/kvm-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
       annotations:
         releaseRevision: {{ .Release.Revision | quote }}
       labels:
-        {{- include "labels.selector" . | nindent 8 }}
+        {{- include "labels.common" . | nindent 8 }}
```
Comment (tfussell) [inline helm/kvm-operator/templates/deployment.yaml:21]: This comes from https://github.com/giantswarm/aws-operator/pull/2451 This allows us to match the pods in okteto based on branch label and also brings kvm-operator in sync with aws-operator's SOTA. It only adds new labels without changing existing ones so I don't expect this to affect anything.

File: Makefile.okteto.mk [hunk 2]
```diff
@@ -0,0 +1,16 @@
+VERSION := $(shell cat pkg/project/project.go | grep version | tr -s ' ' | cut -d ' ' -f 3 | sed 's/"//g')
+VERSION_HYPHENATED := $(shell cat pkg/project/project.go | grep version | tr -s ' ' | cut -d ' ' -f 3 | sed 's/"//g;s/\./-/g')
+
+.PHONY: okteto-up
+## okteto-up: start okteto against the current kubeconfig context
+okteto-up:
```
Comment (tfussell) [inline Makefile.okteto.mk:6]: Automated from https://github.com/giantswarm/aws-operator#live-editing-operator-inside-an-installation

File: README.md [hunk 3]
```diff
@@ -1,14 +1,105 @@
...
+For security issues, please see [the security policy](SECURITY.md).
+
+
+## License
+
+kvm-operator is under the Apache 2.0 license. See the [LICENSE](LICENSE) file
+for details.
+
+
+## Credit
```
Comment (MarcelMue) [inline README.md:103]: Seems like a bit of a random list of credits :D

**Review by MarcelMue — APPROVED** [review]
Haven't tried this okteto setup here but LGTM

**Pull request merged**